waspc/ChangeLog.md

```diff
@@ -28,6 +28,7 @@ Remember to check out the [migration guide](https://wasp.sh/docs/migration-guide
 
 ### 📖 Documentation
 
+- Updated Tailwind CSS installation command in docs from exact version `3.2.7` to `3` (latest v3) to ensure compatibility with Wasp 0.18.0+ ESM config files. ([#3377](https://github.com/wasp-lang/wasp/issues/3377))
```

Comment:
I'd remove this item from the changelog since it is not tied to a specific version. (I know the other items aren't either, but those are from external contributors that we want to make a point of showcasing)